px-alert-message.html

```diff
@@ -126,49 +126,47 @@
         }
       },
     /**
-    * Defines the alert level, reflected in the badge. See [full-screen demo](demo/demo.html) for examples of each.
-    * - 'important' - red triangle labeled 1
-    * - 'warning'- orange diamond labeled 2
-    * - 'error' - yellow square labeled 3
-    * - 'information' - blue circle labeled 4
-    * - 'unknown' - gray circle labeled 5
-    * - 'custom' - white background, no badge
-    * - 'more' - allows for a message indicating that there are more messages in the queue
-    *
-    * @default information
-    *
-    */
+     * Defines the alert level, reflected in the badge. See
+     * [full-screen demo](demo/demo.html) for examples of each.
+     *
+     * The following options are available:
+     *
+     * - 'important' - red triangle labeled 1
+     * - 'warning'- orange diamond labeled 2
+     * - 'error' - yellow square labeled 3
+     * - 'information' - blue circle labeled 4
+     * - 'unknown' - gray circle labeled 5
+     * - 'custom' - white background, no badge
+     * - 'more' - allows for a message indicating that there are more messages in the queue
+     *
+     * @default information
+     */
     type: {
       type: String,
       reflect: true,
-      value: 'information'
+      value: 'information',
+      observer: '_checkMessageLength'
```

Comment:
👍  we moved some things into a complex observer block, and other we chose not to